refiner-lib/src/near_stream.rs

```diff
@@ -160,4 +167,48 @@ mod tests {
         );
         assert_eq!(tx_2.logs.len(), 1);
     }
+
+    #[test]
+    fn test_block_82654651_nonce() {
+        // load state snapshot and main objects
+        let db_dir = tempfile::tempdir().unwrap();
+        let engine_path = db_dir.path().join("engine");
+        let tracker_path = db_dir.path().join("tracker");
+        let chain_id = 1313161554_u64;
+
+        crate::storage::init_storage(engine_path.clone(), "aurora".into(), chain_id);
+        let mut ctx =
+            EngineContext::new(&engine_path, "aurora".parse().unwrap(), chain_id).unwrap();
+
+        let json_snapshot: JsonSnapshot = {
+            let json_snapshot_data = std::fs::read_to_string(
+                "blocks/sate_H7Bfh9qCzWbJW9acao8B2jFMTrkfc31toczmTcMv7hY7.json",
```

Comment:
Maybe it has sense to change the name of the folder and instead of "blocks" call it "test-data" or something else. I think it will be an overkill here to create a inner folder for each test data.